x-pack/filebeat/input/s3/config.go

```diff
@@ -0,0 +1,23 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package s3
+
+import "github.com/elastic/beats/filebeat/harvester"
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "s3",
+	},
+}
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	QueueURLs                 []string `config:"queueURLs" validate:"nonzero,required"`
```

Comment:
we don't use camelcase in user configs, this could be `queue_urls`:

```suggestion
	QueueURLs                 []string `config:"queue_urls" validate:"nonzero,required"`
```